imap_processing/cdf/config/imap_ultra_l1b_variable_attrs.yaml

```diff
@@ -3,210 +3,293 @@ default_attrs: &default
   UNITS: " "
+  dtype: float32
 
 vy_ultra:
-  <<: *default
+  <<: *default_float32
   CATDESC: Normalized component of the velocity vector in y direction in the instrument frame.
   FIELDNAM: vy_ultra
   LABLAXIS: vy ultra
-  # TODO: come back to format
   UNITS: " "
+  dtype: float32
 
 vz_ultra:
-  <<: *default
+  <<: *default_float32
   CATDESC: Normalized component of the velocity vector in z direction in the instrument frame.
   FIELDNAM: vz_ultra
   LABLAXIS: vz ultra
-  # TODO: come back to format
   UNITS: " "
+  dtype: float32
 
 energy:
-  <<: *default
+  <<: *default_float32
   CATDESC: Energy measured using the pulse height from the stop anode (DN) or Energy measured using the SSD (keV) depending on eventtype.
   FIELDNAM: energy
   LABLAXIS: energy
-  # TODO: come back to format
   UNITS: keV
+  dtype: float32
 
 species:
-  <<: *default
-  CATDESC: Species bin.
+  <<: *default_uint8
+  CATDESC: Species bin (0-255).
   FIELDNAM: species
   LABLAXIS: species
-  # TODO: come back to format
   UNITS: " "
+  dtype: uint8
+
+front_back_distance:
+  <<: *default_float32
+  CATDESC: Distance front to back.
+  FIELDNAM: front_back_distance
+  LABLAXIS: front_back_distance
+  UNITS: mm / 100
+  dtype: float32
+
+path_length:
+  <<: *default_float32
+  CATDESC: Path length.
+  FIELDNAM: path_length
+  LABLAXIS: path_length
+  UNITS: mm / 100
+  dtype: float32
 
 event_efficiency:
-  <<: *default
+  <<: *default_float32
   CATDESC: Estimated event efficiency for this path through the instrument.
   FIELDNAM: event_efficiency
   LABLAXIS: event efficiency
-  # TODO: come back to format
   UNITS: " "
+  dtype: float32
 
 vx_sc:
-  <<: *default
-  CATDESC: Normalized component of the velocity vector in x direction in the spacecraft frame.
+  <<: *default_float32
+  CATDESC: x-component of the velocity vector of the ena in the spacecraft frame.
   FIELDNAM: vx_sc
   LABLAXIS: vx sc
-  # TODO: come back to format
   UNITS: " "
+  dtype: float32
 
 vy_sc:
-  <<: *default
-  CATDESC: Normalized component of the velocity vector in y direction in the spacecraft frame.
+  <<: *default_float32
+  CATDESC: y-component of the velocity vector of the ena in the spacecraft frame.
   FIELDNAM: vy_sc
   LABLAXIS: vy sc
-  # TODO: come back to format
   UNITS: " "
+  dtype: float32
 
 vz_sc:
-  <<: *default
-  CATDESC: Normalized component of the velocity vector in z direction in the spacecraft frame.
+  <<: *default_float32
+  CATDESC: z-component of the velocity vector of the ena in the spacecraft frame.
   FIELDNAM: vz_sc
   LABLAXIS: vz sc
-  # TODO: come back to format
   UNITS: " "
+  dtype: float32
 
 vx_dps_sc:
-  <<: *default
-  CATDESC: Normalized component of the velocity vector in x direction in the DPS frame at rest WRT spacecraft (velocity w/ spacecraft frame subtracted)
+  <<: *default_float32
+  CATDESC: x-component of the velocity vector of the ena in the DPS frame at rest WRT spacecraft (velocity w/ spacecraft frame subtracted)
```

Comment:
I'm not sure I understand this description. I understand up to the "velocity vector of the ena in the DPS frame". I don't know what "at rest WRT spacecraft" or "(velocity w/ spacecraft frame subtracted)" are meant to convey.